src/soft-play.js

```diff
@@ -4,11 +4,50 @@ let children = 0
 
 // TODO: Write your functions in the below section. Your functions should update
 // the adults and children variables defined above.
-// Start with the occupancy function.
+// Start with the occupancy function
+let totalAdults = 0
+let totalChildren = 0
+let total=()=>{
+  return {adults : totalAdults , children : totalChildren}
+}
+console.log(total())
+
+ let enter=(numadults , numChildren) =>{
+  if(numadults < numChildren){
+    return false
+  }else{
+    adults += numadults
+    children += numChildren
```

Comment:
Same here. `children` does not exist. You're trying to modify the values of variables that havent been created, so your tests are failing.